Rakefile

```diff
@@ -269,6 +276,10 @@ else
     lib.vendor_lib = "lib/bundler/vendor/thor"
   end
 
+  # We currently cherry-pick changes to use `require_relative` internally
+  # instead of regular `require`. These changes have already landed to fileutils
+  # master, but have not yet been released, so we cherry-pick them for the time
+  # being.
```

Comment:
@hsbt If you have time, it would be nice to release `fileutils` master as fileutils-1.2.1, so we don't need to cherry-pick anything.